package-lock.json

```diff
@@ -3764,7 +3764,7 @@
       "requires": {
```

Comment:
revert this file if not needed.

Reply:
@weidongxu-microsoft 
I'm failing to submit my fixes. Can you please help me out? 

I cloned this repo and made changes in my "copied-from-private"  branch, but cannot push them:

> remote: error: GH006: Protected branch update failed for refs/heads/copied-from-private/avjerafi/azure-rest-api-specs-pr/dev-monitor-Microsoft.Insights-2021-02-01-preview-webHookProperties.
> remote: error: You're not authorized to push to this branch. Visit https://docs.github.com/articles/about-protected-branches/ for more information.
> 

I'm not all familiar with the private+public flow and the documentation I've found does not what steps I need to take in this scenario.

Reply:
That is the reason I asked you to fix in the private repo PR in the first place...

Since the `copied-from-private` is on Azure repo (not your cloned repo), you probably not able to modify it.

Reply:
Sorry, I cannot modify this branch.

You will have to make another branch on your repo (based on this), and make changes on that branch, and send a new PR.

Reply:
Generally, if you do not have any secret changes that does not want non-MSFT to see, you do not need to use the private repo in the first place.

Reply:
Thanks for the input.
If the `copied-from-private` branch is unmodifiable, what am I to do in the current situation where there are discrepancies between the validation outcome in the private vs. the public repo?
I only moved to the public repo because I was under the impression that there will be no fixes required.

- The changes in the `package-lock` file were solely made by running `prettier`.
- The usage of `ResourceGroupNameParameter` passed validation in the private repo.

Reply:
You will have to make another branch on your repo (based on this), and make changes on that branch, and send a new PR.

Reply:
Got it. I'll do that. Thanks. 

What will I have to modify in that new branch so that the `ResourceGroupNameParameter` reference will pass validation?

Reply:
Not sure.

Maybe in the new branch, you can try merge the new changes from master (which include the directive).

If it still not work, you might try delete the following directive from readme.md
```
- from: scheduledQueryRule_API.json
  where: $.parameters
  transform: delete $["ResourceGroupNameParameter"]
  reason: ResourceGroupNameParameter is taken from v2/types.json
```